openassessment/xblock/static/js/src/lms/oa_peer.js

```diff
@@ -16,6 +16,10 @@ OpenAssessment.PeerView = function(element, server, baseView) {
     this.rubric = null;
     this.isRendering = false;
     this.announceStatus = false;
+    this.dateFactory = new OpenAssessment.DateTimeFactory(this.element);
+    this.isRendering = false;
+    this.announceStatus = false;
+
```

Comment:
L20-21 are a repeat of L17-18.

(Same goes for the other files with the apparent "merge conflict" issues.)